source/crud/query/query-filter.txt

```diff
@@ -0,0 +1,633 @@
+
+Geospatial Operators
+--------------------
+
+Geospatial operators return data based on geospatial expression conditions.
+The following table lists the {+driver-short+} methods for
+geospatial operations and the equivalent {+mdb-server+} operators:
+
+.. list-table::
+   :widths: 20 60 20
+   :header-rows: 1
+
+   * - {+driver-short+} Method
+     - Description
+     - {+mdb-server+} Operator
+
+   * - ``GeoIntersects()``
+     - Matches documents whose geospatial data intersects with a specified
+       `GeoJsonObject <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.GeoJsonObjectModel.GeoJsonObject-1.html>`__.
+     - :manual:`$geoIntersects </reference/operator/query/geoIntersects/>`
+
+   * - ``GeoWithin()``
+     - Matches documents whose geospatial data is entirely within the specified shape.
+     - :manual:`$geoWithin </reference/operator/query/geoWithin/>`
+
+   * - ``GeoWithinBox()``
+     - Matches documents whose geospatial data is entirely within the specified box.
+     - :manual:`$geoWithin </reference/operator/query/geoWithin/>`, :manual:`$box </reference/operator/query/box/>`
+
+   * - ``GeoWithinCenter()``
+     - Matches documents whose geospatial data is entirely within the specified circle.
+     - :manual:`$geoWithin </reference/operator/query/geoWithin/>`, :manual:`$center </reference/operator/query/center/>`
+
+   * - ``GeoWithinCenterSphere()``
+     - Matches documents whose geospatial data is entirely within the specified sphere.
+     - :manual:`$geoWithin </reference/operator/query/geoWithin/>`, :manual:`$centerSphere </reference/operator/query/centerSphere/>`
+
+   * - ``GeoWithinPolygon()``
+     - Matches documents whose geospatial data is entirely within the specified polygon.
+     - :manual:`$geoWithin </reference/operator/query/geoWithin/>`, :manual:`$polygon </reference/operator/query/polygon/>`
+
+   * - ``Near()``
+     - Specifies a point for which a geospatial query returns the documents from nearest to
+       farthest.
+     - :manual:`$near </reference/operator/query/near/>`
+
+   * - ``NearSphere()``
+     - Specifies a point for which a geospatial query returns the documents from nearest to
+       farthest in spherical geometry.
+     - :manual:`$nearSphere </reference/operator/query/nearSphere/>`
+
+Bitwise Operators
+-----------------
+
+Bitwise operators matches documents based on bit-position conditions.
+The following table lists the {+driver-short+} methods for
+bitwise operations and the equivalent {+mdb-server+} operators:
+
+.. list-table::
+   :widths: 20 60 20
+   :header-rows: 1
+
+   * - {+driver-short+} Method
+     - Description
+     - {+mdb-server+} Operator
+
+   * - ``BitsAllClear()``
+     - Matches documents where all of the specified bit positions are clear (``0``) in
+       the specified field.
+     - :manual:`$bitsAllClear </reference/operator/query/bitsAllClear/>`
+
+   * - ``bitsAllSet()``
```

Comment:
Initial `b` should be capitalized: `BitsAllSet`

Reply:
good catch.